regolith/helpers/l_membershelper.py

```diff
@@ -68,16 +69,21 @@ def construct_global_ctx(self):
     def sout(self):
         rc = self.rc
         if rc.filter:
-            results = search_collection(self.gtx["people"], rc.filter, rc.keys)
-            print(results, end="")
-            return
+            database = key_value_pair_filter(self.gtx["people"], rc.filter)
```

Comment:
This is just right!   super!

I think this function returns a collection, not a database.  Let's make sure we use the jargon correctly to both avoid confusing people and also training them in the jargon.